README.md

```diff
@@ -189,6 +195,20 @@ Contributing
 
 Please read the [contribution guide](CONTRIBUTING.md) if you want to collaborate in the project.
 
+## Commands
+| Command                                            | Description                                                                                  |
+| -------------------------------------------------- | ---------------------------------------------------------------------------------------------|
+| [skopeo-copy(1)](/docs/skopeo-copy.1.md)           | Copy an image (manifest, filesystem layers, signatures) from one location to another.        |
+| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark image-name for deletion.                                                                |
```

Comment:
Side note, the skopeo-delete(1) page shows a `--force` option in the examples section, but it is not listed as an option in the options section.
Maybe:
```suggestion
| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark the image-name for later deletion by the registry's garbage collector.                                                                |
```

Reply:
The primary user-relevant outcome is that the image may be _immediately_ impossible to pull. GC is a secondary internal implementation details only server administrators care about.

Reply:
FWIW, I picked this and the other point based on info on the respective man pages.  Perhaps they need embellishing too?